README.md

```diff
@@ -127,38 +127,56 @@ in the terminal or copied into the Copy & Paste buffer of your computer.
 Flags:
 
  * `--region <region>`, `-r` -- Specify the `$AWS_DEFAULT_REGION` to use
- * `--arn <arn>`, `-a` -- ARN of role to assume (`$AWS_SSO_ROLE_ARN`)
- * `--account <account>`, `-A` -- AWS AccountID of role to assume (`$AWS_SSO_ACCOUNTID`)
+ * `--arn <arn>`, `-a` -- ARN of role to assume (`$AWS_ROLE_ARN`)
```

Comment:
this variable is used by AWS in some cases... is this safe? https://docs.aws.amazon.com/cli/latest/topic/config-vars.html#assume-role-with-web-identity